BossMod/Network/PacketInterceptor.cs

```diff
@@ -42,8 +42,10 @@ public bool Active
 
         public unsafe PacketInterceptor()
         {
-            // alternative signature: E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 48 8D 35 - seem to be changing from build to build
-            _fetchHook = Service.Hook.HookFromSignature<FetchReceivedPacketDelegate>("E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 44 0F B6 64 24", FetchReceivedPacketDetour);
+            // alternative signatures - seem to be changing from build to build:
+            // - E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 48 8D 35
+            // - E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 44 0F B6 64 24
+            _fetchHook = Service.Hook.HookFromSignature<FetchReceivedPacketDelegate>("E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 48 8D 35", FetchReceivedPacketDetour);
```

Comment:
Huh? What's this?

Reply:
Is it a problem? don't think it will change anything considering it is identical with the existing code

Reply:
No, don't think so, just wondering...